packages/web/src/components/Alert/Alert.scss

```diff
@@ -0,0 +1,53 @@
+@use 'theme';
+@use '../../tools/typography';
+
+.Alert {
+    @include typography.generate(theme.$typography);
+
+    position: relative;
+    display: flex;
+    align-items: center;
+    width: 100%;
+    padding: theme.$space-inner;
+    margin-block: theme.$space-outer;
```

Comment:
From my experience, it's better to **not** have any outer spacing on components because you never know in what context they will be used. Also, there is no outer space on Alert in our Figma source (as opposed to Checkbox component).

Reply:
Fixed.